guides/doc-Release_Notes/topics/foreman-3.8.0.adoc

```diff
@@ -0,0 +1,133 @@
+= Foreman 3.8.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1731[Redmine]
+
+== Foreman
+
+* Include BOOTIF-parameter in kernel_options for Ubuntu autoinstall - https://projects.theforeman.org/issues/36677[#36677]
+* Add LookupValue permissions - https://projects.theforeman.org/issues/36663[#36663]
+* Support Minitest 5.19+ - https://projects.theforeman.org/issues/36651[#36651]
+* Foreman: Open Redirect weakness in links_controller.rb
+ - https://projects.theforeman.org/issues/36644[#36644]
+* The parameter value vanishes when clicking on the hide value checkbox on the new host page paramater tab
+ - https://projects.theforeman.org/issues/36591[#36591]
```

Comment:
Can we continue this to the previous line instead of the new one? The formatting is a bit disturbed with sub-bullets.

Applicable to other places as well.